packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -34,6 +34,12 @@
     white-space: break-word;
   }
 
+  .main-content {
+    @media screen and (max-width: $terra-medium-breakpoint) {
+      min-height: calc(100vh - 120px);
```

Comment:
can you add a comment why we are doing this

Reply:
The help button height is 40px and it has 20px margin top-bottom. Subtracted 80px initially from 100vh to stop the help button sticking on the bottom of the screen. However, even for empty content scroll appears like below
![layout_scroll](https://user-images.githubusercontent.com/21693381/30213288-6c8f194c-9476-11e7-8bbd-90b85442ab74.gif)
Added some extra space to stop scroll when content empty.

Reply:
I mean in the code :)